qiskit/dagcircuit/collect_blocks.py

```diff
@@ -47,6 +47,8 @@ def __init__(self, dag):
         """
 
         self.dag = dag
+        self.pending_nodes = None
+        self.in_degree = None
```

Comment:
```suggestion
        self._pending_nodes = None
        self._in_degree = None
```
Unless these are attributes that would be useful for a user either before or after the pass runs, the convention is to also mark these as "private" (that way users know that they probably shouldn't depend on their existence or values, and we don't need to deprecate them in the future).

Reply:
Thanks, indeed these fields should be private as they do not make much sense after the pass. This and other renaming issues were fixed in [7bc1347](https://github.com/Qiskit/qiskit-terra/pull/8907/commits/7bc1347a2a5c846714924e474b0c4ab4a82e59d7). The only reason I did not commit these suggestions through the web-interface as there were a few more places where renaming had to be done.